.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Should I add `.vscode` to `.gitignore`? Or should I keep this file here? @aldeed

Reply:
I'd keep it so that others using vscode will get these settings